reports/Florida.md

```diff
@@ -35,6 +35,20 @@ id: fl-fortlauderdale-2
 * [CNN Interview](https://twitter.com/greg_doucette/status/1270479109982912514)
 
 
+### Police laugh and joke while "indiscriminately firing" rubber bullets and tear gas | May 31st
```

Comment:
This title needs a little work. First, "indiscriminately firing" is not neutral. Second, it's a nitpick, but it seems that this dialogue happened after firing. It's a small thing but I think there's a difference in degree between while and after

Reply:
Changed the title to focus on the body cams